packages/twenty-front/src/modules/auth/sign-in-up/components/SignInUpGlobalScopeForm.tsx

```diff
@@ -92,9 +92,14 @@ export const SignInUpGlobalScopeForm = () => {
         if (response.__typename === 'UserExists') {
           if (response.availableWorkspaces.length >= 1) {
             const workspace = response.availableWorkspaces[0];
-            return redirectToWorkspaceDomain(workspace.subdomain, pathname, {
-              email: form.getValues('email'),
-            });
+            return redirectToWorkspaceDomain(
+              workspace.subdomain,
```

Comment:
when we have more than one argument, prefer redirectToWorkspaceDomain({subdomain: , ...}) syntax

Reply:
Fix in 424166acfaa2e7dda5653bd450abd25fc9904a10